File: Java/pets.java
import java.nio.channels.AcceptPendingException;

class pets {
    Integer age;
    String size;
    account id;
    account name;
    String race;
    String clinical_history;

    public pets(String race, account id, account name) {
        
    }

    void printDataPet() {
        System.out.println("Race: "+race+", Size: "+size+",Age: "+age);
    }
}

File: JS/pets.js
/*function pets() {
    this.age;
    this.size;
    this.race;
    this.clinical_history;
}*/

class pets {

    constructor (race,size) {
        this.race = race;
        this.size = size;
        this.age;
        this.clinical_history;
    }


    printDataPets() {
        console.log(this.age);
        console.log(this.race);
        console.log(this.size);
        console.log(this.clinical_history);
    }
}